library/core/src/cmp.rs

```diff
@@ -880,7 +880,9 @@ impl PartialOrd for Ordering {
     }
 }
 
-/// Trait for values that can be compared for a sort-order.
+/// Trait for types that form a [partial order](https://en.wikipedia.org/wiki/Partial_order).
+///
+/// Values of types that are `PartialOrd` can be compared for a sort order.
```

Comment:
Calling this a "sort order" is probably more harmful than helpful, given that you can't actually [sort](https://doc.rust-lang.org/1.58.1/std/primitive.slice.html#method.sort) them. Sorting (as used in Rust in the standard library) implies total order. "Topological sort" can handle partial order but is a different thing.

Can you just take this part out? I think the very next sentence showing `<` `<=` etc will give anyone the right impression of what is being talked about.